tests/client/indexer/stream_grpc/test_indexer_grpc_oracle_stream.py

```
@@ -97,6 +97,55 @@ async def test_stream_oracle_prices_by_markets(
+                oracle_type=oracle_type,
+                price=price,
+                timestamp=timestamp,
+            )
+        )
+
+        api = self._api_instance(servicer=oracle_servicer)
+
+        updates = asyncio.Queue()
+        end_event = asyncio.Event()
+
+        callback = lambda update: updates.put_nowait(update)
```

Ruff (0.15.12)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `callback` as a `def`

(E731)